scripts/linux/runE2ETest.sh

```diff
@@ -398,6 +418,10 @@ function process_args() {
     [[ -z "$CONTAINER_REGISTRY_PASSWORD" ]] && { print_error 'Container registry password is required'; exit 1; }
     [[ -z "$IOTHUB_CONNECTION_STRING" ]] && { print_error 'IoT hub connection string is required'; exit 1; }
     [[ -z "$EVENTHUB_CONNECTION_STRING" ]] && { print_error 'Event hub connection string is required'; exit 1; }
+    [[ -z "$ANALYZER_LA_ENABLED_STRING" ]] && { print_error 'Analyzer Log Analytic Enable is required'; exit 1; }
```

Comment:
this should not be required; and other parameter should be required if and only if $ANALYZER_LA_ENABLED_STRING is set to true.

Reply:
Set all of the Log Analytics params to be option, if the params are not define, the `ANALYZER_LA_ENABLED_STRING`, by default, is set to `false`